src/libsync/capabilities.h

```diff
@@ -137,14 +144,14 @@ class OWNCLOUDSYNC_EXPORT Capabilities
     bool uploadConflictFiles() const;
 
     // Direct Editing
-    void addDirectEditor(DirectEditor* directEditor);
-    DirectEditor* getDirectEditorForMimetype(const QMimeType &mimeType);
-    DirectEditor* getDirectEditorForOptionalMimetype(const QMimeType &mimeType);
+    void addDirectEditor(DirectEditor *directEditor);
```

Comment:
Unrelated changes, please avoid those (it's not the only ones I spotted in this patch).